styles.css

```diff
@@ -63,3 +63,40 @@ form {
   font-size: 24pt
 }
 
+.fa-arrow-circle-down,
+.fa-arrow-circle-right {
+  margin-top: 20px;
```

Comment:
Instead of augmenting library classes, just define your own and add them to the component's `className`.